sceptre/sageit/templates/install-studies-bdh-infra.yaml

```diff
@@ -0,0 +1,150 @@
+AWSTemplateFormatVersion: 2010-09-09
+Description: >-
+  Provision a bucket and distribution for install.studies.bridgedigital.health
```

Comment:
i think it would be nice to make this a little more generic by passing in `install.studies.bridgedigital.health` with a parameter.